cpp/Platform.Random/RandomExtensions.h

```diff
@@ -2,10 +2,19 @@
 {
     class RandomExtensions
     {
-        public: static std::uint64_t NextUInt64(System::Random random) { return random.NextUInt64(Range.std::uint64_t); }
+        public: static std::uint64_t NextUInt64(const std::mt19937_64& random)
+        {
+            return std::uniform_int_distribution<std::uint64_t>(Range::UInt64.Minimum, Range::UInt64.Maximum)(random);
```

Comment:
We can use `static const std::uniform_int_distribution` to avoid constructor call